lang/en/docs/workflow-designer/subworkflow-editor/units-flowchart.md

```diff
@@ -2,28 +2,51 @@
 
 At the bottom of the "Overview" tab page within the Subworkflow Editor interface, under the section titled "Units", the user can inspect the **units flowchart** offering a graphical representation of the subworkflow under consideration. Each unit included in this flowchart represents a distinct elementary unit computation, which can be mainly of purely logical (eg. "if" condition) or simulation (eg. ab-initio calculation) nature. The former category is further is narrated further [in its dedicated page](../../workflows/components/units.md).
 
-An example of elementary units flowchart at the bottom of an "Overview" tab, concerning a band structure calculation implemented with the [Quantum ESPRESSO application](../../software-directory/modeling/quantum-espresso/overview.md), is depicted in the image below:  
+An example of elementary units flowchart at the bottom of an "Overview" tab, concerning a band structure and density of states calculation implemented with the [Quantum ESPRESSO application](../../software-directory/modeling/quantum-espresso/overview.md), is depicted in the image below and can be found in the [Materials Bank](https://platform.mat3ra.com/bank/workflows/HPcabYa3gq4BPcb2u):
```

Comment:
It should be `[Workflow Bank](...)`, but you can also just link it above like so:
```markdown
..., concerning a [band structure calculation](...) (with density of states) implemented with the ...
```
IMO either way would be fine